open-banking-accelerator/accelerators/ob-apim/bin/merge.bat

```diff
@@ -0,0 +1,53 @@
+ REM !/bin/bash
+ REM Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ REM
+ REM WSO2 LLC. licenses this file to you under the Apache License,
+ REM Version 2.0 (the "License"); you may not use this file except
+ REM in compliance with the License.
+ REM You may obtain a copy of the License at
+ REM
+ REM    http://www.apache.org/licenses/LICENSE-2.0
+ REM
+ REM Unless required by applicable law or agreed to in writing,
+ REM software distributed under the License is distributed on an
+ REM "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ REM KIND, either express or implied. See the License for the
+ REM specific language governing permissions and limitations
+ REM under the License.
+
+REM merge.bat script copy the WSO2 OB APIM accelerator artifacts on top of WSO2 APIM base product
+REM
+REM merge.bat <APIM_HOME>
```

Comment:
This should be Accelerator Home

Reply:
@Ashi1993 , as discussed offline, this is not required to be changed, right?

Reply:
yes the one in the PR is correct